README.md

```diff
@@ -4,6 +4,10 @@ A frontend for creating and managing email subscriptions.
 
 ## Overview
 
+### Running the application
+
+This application is designed to be run within the [GOV.UK Docker](https://github.com/alphagov/govuk-docker) development environment, adding the `--live` flag
```

Comment:
It looks like we have an unfinished sentence here? 

The `--live` flag doesn't have a bearing on GOV.UK Docker by the way - just the startup.sh script. 

I do still think this should be a separate PR or an issue as it'd be good to get some consensus as any decisions should be things that can be applying across the suite of apps consistently.

Reply:
Maybe a full stop?  Or additional "by'?

> This application is designed to be run within the GOV.UK Docker development environment adding the `--live` flag.
 
I'm just a bit concerned it might get lost and then someone else faces the same issue esp as we have startup documentation information across other FE apps - this feels like an anomaly.

I can raise as an issue? This at least documents it.